invenio_records_files/api.py

```diff
@@ -85,10 +114,22 @@ def dumps(self):
 
 
 def _writable(method):
-    """Check that record is in defined status."""
+    """Check that record is in defined status.
+
+    :param method: Method to be decorated.
+    :returns: Function decorated.
+    """
     @wraps(method)
     def wrapper(self, *args, **kwargs):
-        """Send record for indexing."""
+        """Send record for indexing.
+
+        :param args: List of arguments.
+        :param kwargs: Dictionary of arguments for the function.
+        :returns: Execution result of the decorated method.
+
+        :raises InvalidOperationError: It occurs when the bucket is locked or
+            deleted.
+        """
```

Comment:
I know that this documentation is not showed (Actually not even the ``_writable`` method), but if someone is reviewing the code, it will be helpful.